hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/PlanExecutor.java

```diff
@@ -207,20 +210,28 @@ SqlResult execute(AlterJobPlan plan) {
         if (job == null) {
             throw QueryException.error("The job '" + plan.getJobName() + "' doesn't exist");
         }
-        switch (plan.getOperation()) {
-            case SUSPEND:
-                job.suspend();
-                break;
-
-            case RESUME:
-                job.resume();
-                break;
-
-            case RESTART:
-                job.restart();
-                break;
-
-            default:
+        if (!plan.getOptions().isEmpty()) {
+            JobConfig jobConfig = job.getConfig();
+            processOptions(plan.getOptions(), jobConfig);
```

Comment:
lost update problem is possible here if 2 `ALTER JOB OPTIONS` statements are executed in parallel (e.g. changing different option)

Reply:
I resolved this issue by replacing `void setConfig(JobConfig)` with `JobConfig updateConfig(DeltaJobConfig)`.